Source/Core/DolphinQt2/GameList/TableDelegate.h

```diff
@@ -11,10 +11,13 @@ class TableDelegate final : public QStyledItemDelegate
   Q_OBJECT
 
 public:
-  explicit TableDelegate(QWidget* parent = nullptr);
+  explicit TableDelegate(QWidget* parent);
   void paint(QPainter* painter, const QStyleOptionViewItem& option, const QModelIndex& index) const;
   QSize sizeHint(const QStyleOptionViewItem& option, const QModelIndex& index) const;
 
 private:
-  void DrawPixmap(QPainter* painter, const QRect& rect, const QPixmap& pixmap) const;
+  void DrawPixmap(QPainter* painter, const QRect& rect, const QPixmap& pixmap,
+                  const QSize& size) const;
+  void DrawPixmap(QPainter* painter, const QRect& rect, const QPixmap& pixmap,
```

Comment:
This really confuses me when I'm glancing over the code, can we give these two different names, to reflect what they're each used for?